app/assets/stylesheets/components/_theme.scss

```diff
@@ -13,32 +13,66 @@ $Theme-fontSizes: (
   @return map-get(map-get($Theme-fontSizes, $size), $property);
 }
 
-$Theme-blue: #2E86AB;
-$Theme-greenSuccess: #566612;
-$Theme-orange: #FF623F;
+$Theme-blue: #2A99C8;
+$Theme-beige: #F1EAD9;
 $Theme-white: #FFF;
-$Theme-gray: #EEE;
+$Theme-black: #000;
+$Theme-red: #F56B75;
+$Theme-pink: #E78A91;
+$Theme-yellow: #F6C75C;
+$Theme-darkYellow: #E4B855;
+$Theme-lightGray: #D8D1C2;
+$Theme-gray: #B4B4B4;
+
 $Theme-lightGreenSuccess: #E6F0C0;
+$Theme-greenSuccess: #566612;
 $Theme-lightRedError: #FCE3E4;
 $Theme-redDanger: #BF283A;
 $Theme-redError: #981215;
 
+$Theme-margin-left-logo: 20px;
+
 $Theme-color: (
   background: (
-    default: $Theme-white,
     error: $Theme-lightRedError,
-    success: $Theme-lightGreenSuccess,
-    shadow: $Theme-gray
+    event-card-color: $Theme-beige,
```

Comment:
Also, talk with @fcbaila in order to rename these keys using our left-to-right convention